docker/1.0.0/base/Dockerfile.gpu

```diff
@@ -26,6 +26,7 @@ RUN cd /tmp && \
 
 # Python won’t try to write .pyc or .pyo files on the import of source modules
 # Force stdin, stdout and stderr to be totally unbuffered. Good for logging
-ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 LANG=C.UTF-8 LC_ALL=C.UTF-8
 
-RUN pip install --no-cache-dir Pillow retrying six torch torchvision
+RUN pip install --no-cache-dir Pillow retrying six torch torchvision && \
+    if [ $py_version -eq 3 ]; then pip install --no-cache-dir fastai; fi
```

Comment:
I am not familiar with this part. Why we don't install pytorch in gpu dockerfile?

Reply:
We do, the line 31 above:
```
RUN pip install --no-cache-dir Pillow retrying six torch torchvision 
```

Reply:
So not from binary like CPU?

Reply:
From a different source: in gpu it gets it from pypi, in cpu from a link provided by facebook.